### ISSUE TITLE
Mixed colors come out pale on Mac. They only display correctly under window and cursor shadows.

### ISSUE THREAD

**Issue by atylmo** [pygame/pygame#1214]
I can't capture it with screenshots, but when you do a `surface.fill()` and pass in a color mixture such as for yellow `(255,255,0)` or cyan `(0,255,255)` it looks washed out. Screenshots don't exhibit the problem, nor does accessing the Mac over VNC. It's only visible in person.

Window shadows and mouse cursor shadows that pass over the Pygame window display the color correctly underneath them. If you drag the window down toward the Dock you can see the correct color only within the shadow of it. But primary colors, i.e. red `(255,0,0)`, look fine.

I can temporarily change the monitor's color profile to fix it, but when I start the program again I have the same problem, regardless of which color profile I picked before.

Here's a reduced test case that exhibits the issue:

```
import pygame
from pygame import *

CYAN = (0, 255, 255)

pygame.init()
screen = pygame.display.set_mode((640, 480))
running = True

while running:
    for event in pygame.event.get():
        if event.type == pygame.QUIT:
            pygame.quit()
            sys.exit()
        
    screen.fill(CYAN)
    pygame.display.flip()


```

Specs: Python 3.7.6 via Homebrew, pygame 2.0.0dev3, macOS Catalina Beta 4 (but it also happens on Mojave). I think it also happens with Pygame 1.9.6. It doesn't happen on Ubuntu 19.04.

**Comment by markph0204** [comment]
pygame 2.0.0.dev3 (SDL 2.0.10, python 3.7.3) | Mac 10.14.6

I'm unsure i'm seeing the issue you are describing.  I was only given five crayons as a kid -- so maybe its just my failure to represent colors properly. :)

For the above code -- I was able to use a color picker and get the value: 00ffff

Can you try a color picker and see what result you get? Also try one at least one other Mac?

**Comment by illume** [comment]
Hi,

I guess you get the correct colors in other programs like a web browser?

**Comment by atylmo** [comment]
Yeah, everything else seems fine. I haven't checked it with a color picker yet, though.

**Comment by atylmo** [comment]
This still occurs on the newest Catalina and Pygame 2.0.0-dev4.

**Comment by MyreMylar** [comment]
@atylmo does this still happen for you with 2.0.0.dev8? We haven't had any other reports of this so I'm not sure what to make of it. I wonder if the behaviour is different it you switch to calling display.update() like so:

    import pygame
    import sys
    
    CYAN = (0, 255, 255)
    
    pygame.init()
    screen = pygame.display.set_mode((640, 480))
    running = True
    
    while running:
        for event in pygame.event.get():
            if event.type == pygame.QUIT:
                pygame.quit()
                sys.exit()
    
        screen.fill(CYAN)
        pygame.display.update()

**Comment by eebop** [comment]
Can you you take a screenshot with your computer? If we can use a color picker on that, we can figure out if it's real or just your monitor.

**Comment by Bobbybrojo** [comment]
I had this same issue. Here is a screen shot with the right being the color I want and the left shows the pygame window with slightly muted color. On mac: 
![image](https://user-images.githubusercontent.com/43765519/101469017-06fa2100-38f9-11eb-886d-17d673ade31f.png)
